### ISSUE TITLE
`class` shape, add "stereotype"

### ISSUE THREAD

**Issue by alixander** [terrastruct/d2#1081]
source: https://discord.com/channels/976899413542830181/976909197918756944/1088181820648079380

<img width="542" alt="Screen Shot 2023-03-22 at 5 34 14 PM" src="https://user-images.githubusercontent.com/3120367/227069053-f0305c0d-b071-4473-897d-63894d3a373e.png">

that `<<enumeration>>` thing. UML calls it a "stereotype" but that's a dumb name. i don't really want to introduce another keyword for it though. The only appropriate existing keyword would be "desc", but that's not so fitting. i'll stew on it or wait until someone has a suggestion

**Comment by bo-ku-ra** [comment]
i imagine.
* bullet journal (bullet memo)
 https://www.google.com/search?q=bullet+memo&tbm=isch

**Comment by gourav** [comment]
Hi @alixander 
I would like to take this up. Unfortunately, I don't have a clue how. Can you point me towards right direction to get this done ?

**Comment by alixander** [comment]
i guess this is generalizable to not just `class` shapes

<img width="333" alt="Screenshot 2024-10-16 at 10 23 28 AM" src="https://github.com/user-attachments/assets/ba6061e9-4a0f-40ee-8f2c-dee26f0bbbba">

**Comment by alixander** [comment]
actually you can emulate it sufficiently enough. There's no need for a specific keyword for this.

<img width="701" alt="Image" src="https://github.com/user-attachments/assets/4defc555-0c94-48c5-ab89-0fd5a42d98a5" />

https://play.d2lang.com/?script=rJDBavMwEITv-xSDfU74_56CCIFAH6CHHntR7U29IMlGu8Gkjd-9yHIL7blHfUgz38j3UZLDBwHBv3JwaI7HibOO6XRqCFC7Bd6L-SCdg-Urf8PLmGyn8s4O_w9fAfvEPjvYOO06TsaZgJpXWwAd_MRugyuZpbfB4fBvPW0e52ImatnbmOvD2iohOMyDWDFZaCHSmxrH3ysq_bsVNosZZ4d77Feftm3xXCEB92Ly-IAnn7XcKjLb1C541R9unK6Rszcp3_ySzkHeUuRkDRHQBL5Ysd6qF_oMAAD__w%3D%3D&

Basically the only thing missing is the separate font color for `<<enumeration>>`, but that seems minor
